### PR TITLE
feat: improve build

### PR DESCRIPTION
**Pull request by NikitaZotov**
Update changelog is not needed

### CODE REVIEW

File: scripts/build_platform.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ source "${CURRENT_DIR}/formats.sh"
...
 
-"${CURRENT_DIR}/build_sc_machine.sh"
+"${CURRENT_DIR}/build_sc_machine.sh" "$@"
+"${CURRENT_DIR}/build_sc_component_manager.sh" "$@"
+"${CURRENT_DIR}/build_sc_web.sh" "$@"
 
-"${CURRENT_DIR}/build_sc_component_manager.sh"
-
-stage "OSTIS-web-platform built successfully"
+stage "OSTIS-web-platform is built successfully"
```
Comment (MksmOrlov) [inline scripts/build_platform.sh]:
```suggestion
stage "ostis-web-platform is built successfully"
```

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: scripts/build_platform.sh (hunk 1 above)
Comment (MakarenkoAI) [reply scripts/build_platform.sh]: why?

**Review by MakarenkoAI — APPROVED** [review]

File: scripts/build_platform.sh (hunk 1 above)
Comment (MksmOrlov) [reply scripts/build_platform.sh]: потому что there is no such identifier "OSTIS-web-platform", we have "OSTIS Platform" as standard entity and ostis-web-platform as a repository

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-This repository is a web-oriented software platform of the [OSTIS Technology](https://github.com/ostis-ai) and is intended to be a solid framework to help you deploy existing and create new ostis-systems.
+This repository is a storage of the Platform of the [OSTIS Technology](https://github.com/ostis-ai). Platform OSTIS is intended to be a solid framework to help you deploy existing and create new ostis-systems.
 
-OSTIS Web platform contains:
+OSTIS Platform contains:
 
 1. [Knowledge base with top-level ontologies](https://github.com/ostis-ai/ims.ostis.kb) to help you develop a broad variety of information models
-2. [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and agent-based knowledge processing
+2. [Semantic network storage](https://github.com/ostis-ai/sc-machine) for agent-based knowledge processing
 3. [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) for users to interact with the intelligent system.
```
Comment (MksmOrlov) [inline README.md]: 4. sc-component-manager

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**